showcase/korogenoest.md

```diff
@@ -0,0 +1,130 @@
+
+The raw sequences were initially analysed by the Illumina Dragen pipeline but was moved soon to the Galaxy workflow developed by the Galaxy community led by Wolfgang Maier. The pipeline includes removing the human reads from the raw sequences, mapping the reads to the consensus sequence of SARS-CoV-2, identifying mutations, creating multiple alignments, calling the Pangolin lineages and NextClade clades.  
+
+The work was carried out at the [Galaxy instance](https://galaxy.hpc.ut.ee/) hosted by the [ETAIS](https://etais.ee/) at the [University of Tartu HPC](https://hpc.ut.ee/). All the identified information on the samples was automatically fetched from the dedicated Galaxy address to the Gitlab repository and merged with the rest of the metadata about the samples.
+
+## Generating custom reports
+
+Based on the report templates provided by the ECDC, reports were filled in automatically  based on the sequencing results. These reports included variants of concern (VoCs), specific set of mutations or deletions defined by the ECDC that changed over time. Similar reports were also generated for the Health Board to consult the Government.
+
+## Dashboards
+
+### Description of the closed dashboard
+The data table of all the samples was made available at the access restricted website for the Health Board employees and project participants to provide epidemiological and virological consultations. 
+
+The database included all metadata provided for the samples, sequencing outcomes and was interactive providing filtering and sorting among other customisation options. In addition data could be downloaded for further custom analysis. The closed website of the project dashboard included also different interactive plots that allowed the users to pick the variables to be plotted (e.g. Ct values against vaccination status, county against lineages), but also lineage overview where individual lineages could be included or dismissed, time frame to be selected. 
+
+In addition, for each sequenced plate mutation plots and Ct value plots were implemented to carry out visual inspection as part of the quality control of the sequencing outcomes. In all the dashboards, users could select if they wanted to include data only sequenced in Estonia, by ECDC or both. 
+
+### Public dashboards
+In addition to the access restricted dashboards, public data on the clades, percentage of all positive samples in Estonia been sequenced and clade distribution across counties, regions and age groups were made available and regularly updated at the [Estonian COVID19Dataportal](https://covid19dataportal.ee/genomics_transcriptomics/).  In order to illustrate the virus evolution a public [Nextstrain Auspiece](https://auspice.biit.cs.ut.ee/ncov/est) instance was set up. In regular time intervals, all samples were rerun to obtain the latest lineage annotations.
```

Comment:
```suggestion
In addition to the access restricted dashboards, a regularly updated [Estonian COVID19Dataportal](https://covid19dataportal.ee/genomics_transcriptomics/) public dashboard was also deployed. It made data available across counties, regions and age groups on the clades and clade distribution, and percentage of all positive samples sequenced in Estonia.  In order to illustrate the virus evolution, a public [Nextstrain Auspiece](https://auspice.biit.cs.ut.ee/ncov/est) instance was set up. In regular time intervals, all samples were rerun to obtain the latest lineage annotations.
```

> In regular time intervals, all samples were rerun to obtain the latest lineage annotations

I am not sure what is meant by this. Is it that existing results where processed again?

Reply:
I am not sure what is meant by this. Is it that existing results where processed again?- yes ( lineages change, get renamed etc. this way all the data was always up to date).

Reply:
clarified some wordings